e2e/1_28-exceptions.yml

```diff
@@ -1,11 +1,10 @@
-# This metrics are not present on 1.23
+# This metrics are not present on 1.28
 except_metrics:
 # removed from 1.23
 - k8s.apiserver.etcd.objectCount_*
-# Added from 1.24
-- k8s.controllermanager.nodeCollectorEvictionsDelta
-- k8s.container.oomEventsDelta
 
 # This metrics are pending to be transformed as DM, once they are we can remove this exceptions
 - k8s.apiserver.currentInflightRequestsReadOnly
 - k8s.apiserver.currentInflightRequestsMutating
```

Comment:
Please remove these lines, they are not needed anymore since I fixed them here:
https://source.datanerd.us/infrastructure/infra-integration-specs/pull/320

This comment applies for all exception files

Reply:
Done